Subtitle 1.srt

```diff
@@ -0,0 +1,31 @@
+1
+01:00:18,625 --> 01:00:20,041
+<b>My name is Lucy Nyirenda</b>
+
+2
+01:00:21,208 --> 01:00:23,291
+<b>I stay at Mapalwe Village, Senior group Msiwa</b>

```

Comment:
You'll want to say `live in` instead of `stay at`.

It still isn't clear to me what `Senior group Msiwa` means. Is she a member of Senior group Msiwa? What is Msiwa? I expect a group to have multiple people. The sentence is about where she lives, but this seems to be about her title or rank?